fe/fe-core/src/main/java/org/apache/doris/load/routineload/RoutineLoadJob.java

```diff
@@ -789,6 +789,12 @@ private void updateNumOfData(long numOfTotalRows, long numOfErrorRows, long unse
         this.jobStatistic.unselectedRows += unselectedRows;
         this.jobStatistic.receivedBytes += receivedBytes;
         this.jobStatistic.totalTaskExcutionTimeMs += taskExecutionTime;
+        if (currentTaskConcurrentNum > 0) {
+            this.jobStatistic.taskExecuteTimeMs = this.jobStatistic.totalTaskExcutionTimeMs / currentTaskConcurrentNum;
```

Comment:
Can this.jobStatistic.totalTaskExcutionTimeMs / currentTaskConcurrentNum calculate accurately？

Reply:
Not absolutely accurate, but relatively close. The best way to calculate this is to add up the maximum execution times of multiple concurrent instances each time, but I haven't found a specific way to do this. I'm willing to close my PR if someone has a better way to implement it.

Reply:
How about add a timer to the job to calculate the time?

Reply:
Theoretically a good idea， I'll try it. Thanks.